src/lib.rs

```diff
@@ -842,9 +842,9 @@ macro_rules! quote_token_with_context_spanned {
 }
 
 // These rules are ordered by approximate token frequency, at least for the
-// first 10 or so, to improve compile times. Having `ident` first is by far the
-// most important, because it's typically 2-3x more common than the next most
-// common token.
```

Comment:
This comment shouldn't have been removed. The language was a bit unclear, it was about the `ident` *rule* being the first rule, rather than the `ident` being the first token within the rule.

Reply:
I'll fix it